package.json

```diff
@@ -26,86 +26,84 @@
   },
   "homepage": "https://github.com/Financial-Times/n-ui#readme",
   "devDependencies": {
-    "@financial-times/n-gage": "^1.8.25",
-    "@financial-times/n-heroku-tools": "^6.28.6",
-    "bower": "^1.8.0",
-    "bundlesize": "^0.13.2",
-    "chai": "^3.4.0",
+    "@financial-times/n-gage": "^1.9.5",
+    "@financial-times/n-heroku-tools": "^6.30.0",
+    "bower": "^1.8.2",
+    "bundlesize": "^0.15.2",
+    "chai": "^4.1.2",
     "chalk": "^2.1.0",
-    "coveralls": "^2.13.1",
+    "coveralls": "^3.0.0",
     "denodeify": "^1.2.1",
     "eslint": "^2.11.1",
     "eslint-plugin-react": "^5.1.1",
-    "fetch-mock": "^5.12.1",
+    "fetch-mock": "^5.13.0",
     "handlebars": "^4.0.10",
-    "iltorb": "^1.3.5",
+    "iltorb": "^1.3.9",
     "istanbul": "^0.4.5",
-    "karma": "^1.7.0",
+    "karma": "^1.7.1",
     "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-firefox-launcher": "^1.0.1",
     "karma-html-reporter": "^0.2.7",
     "karma-mocha": "^1.3.0",
-    "karma-sauce-launcher": "^1.0.0",
+    "karma-sauce-launcher": "^1.2.0",
     "karma-sinon": "^1.0.5",
-    "karma-sinon-chai": "^1.2.0",
+    "karma-sinon-chai": "^1.3.2",
     "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^2.0.4",
+    "karma-webpack": "^2.0.5",
     "lintspaces-cli": "^0.6.0",
     "mocha": "^3.5.0",
     "mockery": "^2.1.0",
     "nightwatch": "^0.9.16",
-    "node-fetch": "^1.7.2",
-    "nodemon": "^1.11.0",
-    "npm-prepublish": "^1.2.2",
+    "node-fetch": "^1.7.3",
+    "nodemon": "^1.12.1",
+    "npm-prepublish": "^1.2.3",
     "pa11y-ci": "^1.2.0",
     "proxyquire": "^1.8.0",
     "semver": "^5.4.1",
-    "sinon": "^1.17.7",
-    "sinon-chai": "^2.12.0",
+    "sinon": "^4.0.1",
+    "sinon-chai": "^2.14.0",
     "supertest": "^3.0.0",
     "uglifyjs": "^2.4.11"
   },
   "dependencies": {
-    "@financial-times/n-desktop-app-banner": "^1.0.1",
-    "@financial-times/n-express": "^19.2.3",
+    "@financial-times/n-desktop-app-banner": "^1.0.9",
+    "@financial-times/n-express": "^19.3.0",
     "@financial-times/n-handlebars": "^1.17.9",
     "@financial-times/next-json-ld": "^0.2.1",
-    "autoprefixer": "^7.1.2",
-    "aws-sdk": "^2.95.0",
-    "babel-core": "^6.25.0",
-    "babel-loader": "^7.1.1",
+    "autoprefixer": "^7.1.5",
+    "aws-sdk": "^2.130.0",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
     "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-transform-es2015-classes": "^6.14.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.8.0",
+    "babel-plugin-transform-es2015-classes": "^6.24.1",
```

Comment:
can these two deps be removed now we're on preset-env?

Reply:
@wheresrhys `modules-commonjs` can be removed (unused). Provided there is a way to configure `transform-es2015-classes` to `loose` in the preset flags, I'll remove `babel-plugin-transform-es2015-classes` too. Agreed on the beta release.

Reply:
re. loose mode - as IE10 should now be considered a core browser I assume we can drop it? https://babeljs.io/docs/usage/caveats/#internet-explorer-classes-10-and-below-

Reply:
thanks, @i-like-robots – I'm for dropping it